.lock-waf_darwin_build

```diff
@@ -0,0 +1,8 @@
+argv = ['/Users/orviwan/Pebble/tintin/build/sdk/waf', 'configure']
```

Comment:
Don't commit this unless you like sharing all your environment variables...looks like this project is missing a recent/complete .gitignore file. Try the SDK default one